LoopUI/Charts/PredictedGlucoseChart.swift

```diff
@@ -245,10 +245,46 @@ extension PredictedGlucoseChart {
     }
 
     public func setPredictedGlucoseValues(_ glucoseValues: [GlucoseValue]) {
-        predictedGlucosePoints = glucosePointsFromValues(glucoseValues)
+        let sanitizedPredictedGlucoseValues = sanitizePredictedGlucoseValues(glucoseValues)
+        predictedGlucosePoints = glucosePointsFromValues(sanitizedPredictedGlucoseValues)
     }
 
     public func setAlternatePredictedGlucoseValues(_ glucoseValues: [GlucoseValue]) {
         alternatePredictedGlucosePoints = glucosePointsFromValues(glucoseValues)
     }
 }
+
+
+// MARK: - Clipping the predicted glucose values to a suggested min and max
+import HealthKit
+
+extension PredictedGlucoseChart {
+    var suggestedPredictedGlucoseMax: HKQuantity {
+        return HKQuantity(unit: .milligramsPerDeciliter, doubleValue: 400)
+    }
+    
+    var suggestedPredictedGlucoseMin: HKQuantity {
+        return HKQuantity(unit: .milligramsPerDeciliter, doubleValue: 0)
+    }
+    
+    var maxChartedGlucoseValue: HKQuantity? {
```

Comment:
[nit] Remove "Charted"? Both GlucoseValues and PredictedGlucoseValues are eventually charted.